yt_dlp/extractor/clyp.py

```diff
@@ -74,3 +97,11 @@ def _real_extract(self, url):
             'timestamp': timestamp,
             'formats': formats,
         }
+
+    def get_wav_format(self, url, audio_id):
+        page = self._download_webpage(url, video_id=audio_id)
+        wav_url = self._html_search_regex(
+            r'var wavStreamUrl = \"(?P<url>.+)\"', page, 'url', default='')
+
+        if wav_url.startswith('https://'):
+            return wav_url
```

Comment:
* Doesn't need to be a function
* The https check can be part of the regex
* Relax the regex with `\s*`
* Change `\"` to `\'"`, `.+` to `[^\'"]+`